src/query-builder/components/TextField.tsx

```diff
@@ -23,10 +18,12 @@ const TextField: FC<{
     if (trimmed.length) {
       if (field.valuePrefix && field.term === 'xref' && trimmed === '*') {
         // Query is faster with this hack
-        handleChange({ xref: `${field.valuePrefix}` });
+        // NOTE: Is this even working?..
```

Comment:
You can remove that "optimisation hack" as it's actually not working...it should use `database` instead. Let's just remove and do `xref: myxref-*`

Reply:
Shoudn't that change be part of the jira you took this morning anyway?

Reply:
I don't understand how xref queries are supposed to work anyway, especially because I haven't seen any working fine yet, so it's hard to implement against...

Reply:
Ok merge it and I’ll take it from there